<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图片提取器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>761</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请选择该链接内容的类型</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="s2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>鱼C主题</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="s1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>B站文章</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="s3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CSDN博客</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="s4">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>70</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>其他</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="css">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>使用自定义 CSS 选择器</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="exp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>511</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="doac">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>110</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>提取</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>761</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="confirm">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>110</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="url">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>109</y>
       <width>501</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>请输入链接</string>
     </property>
    </widget>
    <widget class="QLabel" name="label3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>错误的链接</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>761</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>目前下载进度</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="dpg">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="stop">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>40</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
    </widget>
    <widget class="QPushButton" name="copyua">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>40</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>复制 UA</string>
     </property>
    </widget>
    <widget class="QLabel" name="label5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>下载进度</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
